### PR TITLE
Create osget.bat

### PR DESCRIPTION
**Pull request by Teknoist**
can your help me to converting your tool for windows ?

### CODE REVIEW

**Comment by gnarlin** [comment]
I'm sorry, but no. I definitely do NOT want to re-write osget in something that's specific to only Microsoft Windows.
I do want to eventually re-write Osget in Python3 and then eventually make it also platform aware (ie. check in which operating system it was being run and then change all the paths and capital awareness shenanigans that probably have to be done) but I already want to do other things (change the repositories to use git instead of ftp and to create automagical distro version scrapers) and I'm already procrastinating on those ideas. 
Bottom line, if you want to re-write Osget in DOS batch script or even powershell then knock yourself out but I would much prefer for Osget to eventually be ported to Python and have a single code base that's platform agnostic/platform aware.

**Comment by Teknoist** [comment]
Your solution is much smarter, I hadn't thought of that 😅
